### ISSUE TITLE
 villagers not linking to work stations

### ISSUE THREAD

**Issue by ezzle1945** [Iskallia/Vault-public#932]
when i pick up a villager and move it even if its like over 300 blocks the villager doesnt link to a new work station and will always become a farmer even though no composters are around and even if i put composters down they dont link so trades never reset.

**Comment by chico1992** [comment]
They need to unlink from their workstation before you pick them up else they'll stay linked to their old workstation and will try to pathfind to it for a long time

**Comment by wotupfoo** [comment]
Agreed. You have to break their original workstation - eg composter for farmer back at their village. 
3000+ blocks back at home I learned that the hard way.
